addons/main/functions/fnc_hasInjector.sqf

```diff
@@ -0,0 +1,12 @@
+#include "script_component.hpp"
+params ["_unit"];
+
+private _items = call FUNC(uniqueItems);
```

Comment:
Variable assignment is fairly slow in SQF, if the variable is used only once it is considered a bit of a waste in run time. (real talk, it is almost negligible, but trying to optimize small bits help overall)

Combined with the part the variable is being read from, in this case the `uniqueItems` call, you could just directly use it in line 6, as it is short enough to be still readable.